tests/layer_tests/common/utils/common_utils.py

```diff
@@ -25,6 +25,8 @@ def generate_ir(coverage=False, **kwargs):
             params.extend(("-b", str(value)))
         elif key == "k":
             params.extend(("-k", str(value)))
+        elif key == "use_new_frontend":
```

Comment:
no need in new key, it should be correctly handled on 30 line (if specify it as {"use_new_frontend": True})